packages/create-sitecore-jss/src/bin.ts

```diff
@@ -26,6 +36,60 @@ export const parseArgs = (): ParsedArgs => {
   return args;
 };
 
+export const getDestinations = async (args: ParsedArgs, templates: string[]) => {
+  if (templates.length === 0) {
+    throw new Error('Unable to get destinations, provided templates are empty');
+  }
+  // validate/gather destination
+  const defaultBaseDestination = `${process.cwd()}${
+    args.appName ? sep + args.appName : `${sep}${templates[0]}`
+  }`;
+  const destination = args.destination
+    ? args.destination
+    : args.yes
+    ? defaultBaseDestination
+    : await promptDestination('Where would you like your new app created?', defaultBaseDestination);
```

Comment:
Should we use regular if/else for the ease of reading this?

Reply:
I can switch to if/else. The way it's done now was to not have too large of a duplicating code block (same logic is used further down)